app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  enum status: %i[unprocessed approved rejected]
+
+  validates :rating, numericality: { greater_than_or_equal_to: 1, less_than_or_equal_to: 5 }
```

Comment:
Can we use this `inclusion`? https://guides.rubyonrails.org/active_record_validations.html#inclusion